controller/uninstall_controller.go

```diff
@@ -35,7 +35,9 @@ const (
 	CRDEngineImageName     = "engineimages.longhorn.io"
 	CRDNodeName            = "nodes.longhorn.io"
 	CRDInstanceManagerName = "instancemanagers.longhorn.io"
+	CRDShareManagerName    = "sharemanagers.longhorn.io"
 	CRDBackingImageName    = "backingimages.longhorn.io"
+	CRDSettingName         = "settings.longhorn.io"
```

Comment:
I think we don't need to handle `settings` resources for now because in the current implementation, we don't need any special logic for handling the settings cleanup via finalizer, but other resources all managed by finalizer to make sure we can clean up the corresponding pods correctly.

But, I don't think we need to have that for `node` as well, but this was implemented already. Maybe we can clean up uninstaller in near future, also many duplicated code.

Reply:
removed